### PR TITLE
fix(login): better server list design

### PR DESCRIPTION
**Pull request by lukas-mertens**
Signed-off-by: Lukas Mertens <git@lukas-mertens.de>

---

**Stack**:
- #86
- #83
- #78
- #70
- #68
- #67
- #64
- #63
- #62
- #61
- #60
- #58 ⬅

⚠️ *Part of a stack created by [spr](https://github.com/ejoffe/spr). Do not merge manually using the UI - doing so may have unexpected results.*

### CODE REVIEW

**Comment by lukas-mertens** [comment]
## Before


## After

File: src/pages/LoginPage.vue [hunk 1]
```diff
@@ -68,15 +68,17 @@
                   </v-container>
                 </v-form>
                 <template v-else>
-                  <v-list-subheader lines="two" :disabled="connecting">
-                    <v-list-item append-icon="mdi-server" v-for="(server, index) in servers" :key="server.id" @click="connect(server.addr)">
+                  <v-list-subheader lines="two" :disabled="connecting" class="mb-3">
+                    <v-list-item prepend-icon="mdi-server" v-for="(server, index) in servers" :key="server.id" @click="connect(server.addr)">
```
Comment (felix-ulonska) [inline src/pages/LoginPage.vue:73]:
```suggestion
```

File: src/pages/LoginPage.vue [hunk 2]
```diff
@@ -68,15 +68,17 @@
...
 
                         <v-list-item-title>{{ server.id }}</v-list-item-title>
                         <v-list-item-subtitle>{{ server.addr }}</v-list-item-subtitle>
-                      
-                      <v-list-item-action v-if="server.editable">
-                        <v-btn icon="mdi-pencil" @click.prevent.stop="edit_server(index)"></v-btn>
-                      </v-list-item-action>
+
```
Comment (felix-ulonska) [inline src/pages/LoginPage.vue:76]:
```suggestion
```

**Review by felix-ulonska — APPROVED** [review]
Except for the two white suggestions, approve

**Comment by lukas-mertens** [comment]
✓ Commit merged in pull request [#70](https://github.com/EVerest/everest-admin-panel/pull/70)

**Pull request closed without merging**